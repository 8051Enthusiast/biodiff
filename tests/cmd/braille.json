{
  "version": "1",
  "presets": {
    "global": [
      {
        "name": "default",
        "gap_open": -5,
        "gap_extend": -1,
        "mismatch_score": -1,
        "match_score": 0,
        "mode": {
          "Blockwise": 8192
        },
        "backend": {
          "backend": "wfa2"
        }
      },
      {
        "name": "global",
        "gap_open": -5,
        "gap_extend": -1,
        "mismatch_score": -1,
        "match_score": 0,
        "mode": "Global",
        "backend": {
          "backend": "wfa2"
        }
      }
    ],
    "current_global": 0,
    "semiglobal": [
      {
        "name": "default",
        "gap_open": -5,
        "gap_extend": -1,
        "mismatch_score": -1,
        "match_score": 0,
        "mode": "Semiglobal",
        "backend": {
          "backend": "wfa2"
        }
      }
    ],
    "current_semiglobal": 0
  },
  "style": {
    "mode": "Braille",
    "ascii_col": true,
    "bars_col": false,
    "layout": "Horizontal",
    "spacer": false,
    "right_to_left": false,
    "column_count": {
      "Fixed": 24
    },
    "no_scroll": false
  }
}