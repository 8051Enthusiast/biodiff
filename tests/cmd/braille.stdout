[38;5;8m 0000 [39mâ”‚â¡’â”‚â Žâ”‚â¢Žâ”‚â£¦â”‚â „â”‚â¢Žâ”‚â£¦â”‚â „â”‚â£¦â”‚â¢¾â”‚â¢žâ”‚â¢–â”‚â „â”‚â¢¦â”‚â¢¾â”‚â¢žâ”‚â¢žâ”‚â¢¾â”‚â ¾â”‚â „â”‚â¡–â”‚â¢–â”‚â¡Žâ”‚â¡–[38;5;8m |[39mThis is some common text[38;5;8m | 0000 [39mâ”‚â¡’â”‚â Žâ”‚â¢Žâ”‚â£¦â”‚â „â”‚â¢Žâ”‚â£¦â”‚â „â”‚â£¦â”‚â¢¾â”‚â¢žâ”‚â¢–â”‚â „â”‚â¢¦â”‚â¢¾â”‚â¢žâ”‚â¢žâ”‚â¢¾â”‚â ¾â”‚â „â”‚â¡–â”‚â¢–â”‚â¡Žâ”‚â¡–[38;5;8m |[39mThis is some common text
[38;5;8m 0018 [38;5;11mâ”‚â „â”‚â ¦â”‚â¢–â”‚â¡–â”‚â£¶â”‚â¢–â”‚â¢–â”‚â ¾â”‚â „â”‚â ¦â”‚â¢¾â”‚â¡–â”‚â Žâ”‚â „â”‚â ¶â”‚â¢Žâ”‚â žâ”‚â¢–â”‚â£¦â”‚â ¼[38;5;2mâ”‚â „â”‚â¡’â”‚â Žâ”‚â¢Ž[38;5;8m |[38;5;11m between both files.[38;5;2m Thi[38;5;8m | 0018 [38;5;11mâ”‚â „â”‚â ¦â”‚â¢–â”‚â¡–â”‚â£¶â”‚â¢–â”‚â¢–â”‚â ¾â”‚â „â”‚â ¦â”‚â¢¾â”‚â¡–â”‚â Žâ”‚â „â”‚â ¶â”‚â¢Žâ”‚â žâ”‚â¢–â”‚â£¦â”‚â ¼[38;5;2m        [38;5;8m |[38;5;11m between both files.[38;5;2m    [39m
[38;5;8m 0030 [38;5;10mâ”‚â£¦â”‚â „â”‚â¡–â”‚â¢–â”‚â¡Žâ”‚â¡–â”‚â „â”‚â¢Žâ”‚â£¦â”‚â „â”‚â¢¾â”‚â ¾â”‚â žâ”‚â£Žâ”‚â „â”‚â¢Žâ”‚â ¾â”‚â „â”‚â¡–â”‚â Žâ”‚â¢Žâ”‚â£¦â”‚â „â”‚â ¶[38;5;8m |[38;5;10ms text is only in this f[38;5;8m | 002c [38;5;10m                                                [38;5;8m |[38;5;10m                        [39m
[38;5;8m 0048 [38;5;2mâ”‚â¢Žâ”‚â¡¦â”‚â£¦â”‚â¡–â”‚â „â”‚â ¶â”‚â¢Žâ”‚â žâ”‚â¢–â”‚â ¼[38;5;11mâ”‚â ¨â”‚â¡’â”‚â Žâ”‚â¢–â”‚â „â”‚â¢¦â”‚â Žâ”‚â¢†â”‚â¡¦â”‚â¢†â”‚â¢¦â”‚â¡–â”‚â¢–â”‚â¡¦[38;5;8m |[38;5;2mirst file.[38;5;11m.The character[38;5;8m | 002c [38;5;2m                    [38;5;11mâ”‚â ¨â”‚â¡’â”‚â Žâ”‚â¢–â”‚â „â”‚â¢¦â”‚â Žâ”‚â¢†â”‚â¡¦â”‚â¢†â”‚â¢¦â”‚â¡–â”‚â¢–â”‚â¡¦[38;5;8m |[38;5;2m          [38;5;11m.The character[39m
[38;5;8m 0060 [39mâ”‚â „â”‚â¢Žâ”‚â£¦â”‚â „[38;5;9mâ”‚â¡Š[39mâ”‚â „â”‚â¢Žâ”‚â ¾â”‚â£¦â”‚â¡–â”‚â¢–â”‚â¢†â”‚â –â”‚â „â”‚â¢¾â”‚â ¶â”‚â „[38;5;9mâ”‚â£Š[39mâ”‚â ¼â”‚â ¨â”‚â£¢â”‚â¢¾â”‚â¢žâ”‚â¢–[38;5;8m |[39m is [38;5;9mX[39m instead of [38;5;9mY[39m..Some[38;5;8m | 003a [39mâ”‚â „â”‚â¢Žâ”‚â£¦â”‚â „[38;5;9mâ”‚â£Š[39mâ”‚â „â”‚â¢Žâ”‚â ¾â”‚â£¦â”‚â¡–â”‚â¢–â”‚â¢†â”‚â –â”‚â „â”‚â¢¾â”‚â ¶â”‚â „[38;5;9mâ”‚â¡Š[39mâ”‚â ¼â”‚â ¨â”‚â£¢â”‚â¢¾â”‚â¢žâ”‚â¢–[38;5;8m |[39m is [38;5;9mY[39m instead of [38;5;9mX[39m..Some
[38;5;8m 0078 [38;5;11mâ”‚â „â”‚â¢žâ”‚â¢¾â”‚â¡¦â”‚â¢–â”‚â „â”‚â¡–â”‚â¢–â”‚â¡Žâ”‚â¡–â”‚â „â”‚â¡–â”‚â¢¾â”‚â „â”‚â¢žâ”‚â¢†â”‚â¢®â”‚â¢–â”‚â „â”‚â£¦â”‚â£–â”‚â¡¦â”‚â¢–â”‚â „[38;5;8m |[38;5;11m more text to make sure [38;5;8m | 0052 [38;5;11mâ”‚â „â”‚â¢žâ”‚â¢¾â”‚â¡¦â”‚â¢–â”‚â „â”‚â¡–â”‚â¢–â”‚â¡Žâ”‚â¡–â”‚â „â”‚â¡–â”‚â¢¾â”‚â „â”‚â¢žâ”‚â¢†â”‚â¢®â”‚â¢–â”‚â „â”‚â£¦â”‚â£–â”‚â¡¦â”‚â¢–â”‚â „[38;5;8m |[38;5;11m more text to make sure [39m
[38;5;8m 0090 [39mâ”‚â¡–â”‚â Žâ”‚â¢–â”‚â „â”‚â ¾â”‚â£–â”‚â¢žâ”‚â ¦â”‚â¢–â”‚â¡¦â”‚â „â”‚â¢¾â”‚â ¶â”‚â „â”‚â£¦â”‚â¢†â”‚â¢žâ”‚â¢–â”‚â „â”‚â¢¦â”‚â Žâ”‚â¢†â”‚â¡¦â”‚â¢†[38;5;8m |[39mthe number of same chara[38;5;8m | 006a [39mâ”‚â¡–â”‚â Žâ”‚â¢–â”‚â „â”‚â ¾â”‚â£–â”‚â¢žâ”‚â ¦â”‚â¢–â”‚â¡¦â”‚â „â”‚â¢¾â”‚â ¶â”‚â „â”‚â£¦â”‚â¢†â”‚â¢žâ”‚â¢–â”‚â „â”‚â¢¦â”‚â Žâ”‚â¢†â”‚â¡¦â”‚â¢†[38;5;8m |[39mthe number of same chara
[38;5;8m 00a8 [38;5;11mâ”‚â¢¦â”‚â¡–â”‚â¢–â”‚â¡¦â”‚â£¦â”‚â „â”‚â ¦â”‚â¢–â”‚â¡–â”‚â£¶â”‚â¢–â”‚â¢–â”‚â ¾â”‚â „â”‚â¡–â”‚â Žâ”‚â¢–â”‚â „â”‚â¡–â”‚â¢–â”‚â¡Žâ”‚â¡–â”‚â£¦â”‚â „[38;5;8m |[38;5;11mcters between the texts [38;5;8m | 0082 [38;5;11mâ”‚â¢¦â”‚â¡–â”‚â¢–â”‚â¡¦â”‚â£¦â”‚â „â”‚â ¦â”‚â¢–â”‚â¡–â”‚â£¶â”‚â¢–â”‚â¢–â”‚â ¾â”‚â „â”‚â¡–â”‚â Žâ”‚â¢–â”‚â „â”‚â¡–â”‚â¢–â”‚â¡Žâ”‚â¡–â”‚â£¦â”‚â „[38;5;8m |[38;5;11mcters between the texts [39m
[38;5;8m 00c0 [39mâ”‚â¢Žâ”‚â£¦â”‚â „â”‚â£¦â”‚â£–â”‚â ¶â”‚â ¶â”‚â¢Žâ”‚â¢¦â”‚â¢Žâ”‚â¢–â”‚â ¾â”‚â¡–â”‚â žâ”‚â£Žâ”‚â „â”‚â Žâ”‚â¢Žâ”‚â¢¶â”‚â Žâ”‚â ¼â”‚â ¨[38;5;10m    [38;5;8m |[39mis sufficiently high..[38;5;10m  [38;5;8m | 009a [39mâ”‚â¢Žâ”‚â£¦â”‚â „â”‚â£¦â”‚â£–â”‚â ¶â”‚â ¶â”‚â¢Žâ”‚â¢¦â”‚â¢Žâ”‚â¢–â”‚â ¾â”‚â¡–â”‚â žâ”‚â£Žâ”‚â „â”‚â Žâ”‚â¢Žâ”‚â¢¶â”‚â Žâ”‚â ¼â”‚â ¨[38;5;10mâ”‚â¡’â”‚â Ž[38;5;8m |[39mis sufficiently high..[38;5;10mTh[39m
[38;5;8m 00d6 [38;5;2m                                                [38;5;8m |[38;5;2m                        [38;5;8m | 00b2 [38;5;2mâ”‚â¢Žâ”‚â£¦â”‚â „â”‚â¡–â”‚â¢–â”‚â¡Žâ”‚â¡–â”‚â „â”‚â¢Žâ”‚â£¦â”‚â „â”‚â¢¾â”‚â ¾â”‚â žâ”‚â£Žâ”‚â „â”‚â¢Žâ”‚â ¾â”‚â „â”‚â¡–â”‚â Žâ”‚â¢Žâ”‚â£¦â”‚â „[38;5;8m |[38;5;2mis text is only in this [39m
[38;5;8m 00d6 [38;5;10m                          [39mâ”‚â Šâ”‚â¢–â”‚â¡¦â”‚â¢–â”‚â „â”‚â¢Žâ”‚â£¦â”‚â „â”‚â£¦â”‚â¢¾â”‚â¢ž[38;5;8m |[38;5;10m             [39mHere is som[38;5;8m | 00ca [38;5;10mâ”‚â£¦â”‚â¢–â”‚â¢¦â”‚â¢¾â”‚â ¾â”‚â –â”‚â „â”‚â ¶â”‚â¢Žâ”‚â žâ”‚â¢–â”‚â ¼â”‚â „[39mâ”‚â Šâ”‚â¢–â”‚â¡¦â”‚â¢–â”‚â „â”‚â¢Žâ”‚â£¦â”‚â „â”‚â£¦â”‚â¢¾â”‚â¢ž[38;5;8m |[38;5;10msecond file. [39mHere is som
[38;5;8m 00e1 [38;5;11mâ”‚â¢–â”‚â „â”‚â¢žâ”‚â¢¾â”‚â¡¦â”‚â¢–â”‚â „â”‚â¢¦â”‚â¢¾â”‚â¢žâ”‚â¢žâ”‚â¢¾â”‚â ¾â”‚â „â”‚â¡–â”‚â¢–â”‚â¡Žâ”‚â¡–â”‚â „â”‚â¡–â”‚â¢¾â”‚â „â”‚â£¶â”‚â¡¦[38;5;8m |[38;5;11me more common text to wr[38;5;8m | 00e2 [38;5;11mâ”‚â¢–â”‚â „â”‚â¢žâ”‚â¢¾â”‚â¡¦â”‚â¢–â”‚â „â”‚â¢¦â”‚â¢¾â”‚â¢žâ”‚â¢žâ”‚â¢¾â”‚â ¾â”‚â „â”‚â¡–â”‚â¢–â”‚â¡Žâ”‚â¡–â”‚â „â”‚â¡–â”‚â¢¾â”‚â „â”‚â£¶â”‚â¡¦[38;5;8m |[38;5;11me more common text to wr[39m
[38;5;8m 00f9 [39mâ”‚â¢†â”‚â¡†â”‚â „â”‚â¢Žâ”‚â¡–â”‚â „â”‚â£–â”‚â¡†â”‚â ¼[38;5;10m                              [38;5;8m |[39map it up.[38;5;10m               [38;5;8m | 00fa [39mâ”‚â¢†â”‚â¡†â”‚â „â”‚â¢Žâ”‚â¡–â”‚â „â”‚â£–â”‚â¡†â”‚â ¼[38;5;10m                              [38;5;8m |[39map it up.[38;5;10m               [39m
